core/src/main/resources/lib/form/rowSet.jelly

```diff
@@ -35,12 +35,6 @@ THE SOFTWARE.
     <st:attribute name="ref">
       id of the thing that serves as the group head, if that's available separately
     </st:attribute>
-    <st:attribute name="isFormItem">
```

Comment:
This attribute isn't being used, so removed it.

https://github.com/search?q=org%3Ajenkinsci%20isFormItem%3D&type=code